packages/react-composites/src/composites/ChatComposite/index.ts

```diff
@@ -34,16 +34,15 @@ export type { ResourceDetails } from './adapter/ChatAdapter';
 export * from './Strings';
 
 /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
```

Comment:
Is this CC flag still needed?

Reply:
The options are still needed for download attachment (since have not GA), and upload assumes it needs everything in download